### ISSUE TITLE
sections=... order isn't obeyed

### ISSUE THREAD

**Issue by w0rp** [timothycrosley/isort#289]
I defined a configuration file like so for trying out isort with my website code.

```
[settings]
line_length=79
multi_line_output=3
include_trailing_comma=1
sections=FUTURE,STDLIB,THIRDPARTY,W0RPLIB,BLOG,FIRSTPARTY,LOCALFOLDER
known_third_party=django
known_blog=blog
known_w0rplib=w0rplib
```

So I can force all of the `blog` imports to group together in one block, and all of the `w0rplib` imports to group together above those.

Then I tried it out against a Python file with these imports, specified in the order I want:

```
from __future__ import unicode_literals

import datetime

from django.conf import settings

from w0rplib.templatetags.markdown import markdown

from blog.util import whatever

from .managers import ArticleManager
```

`isort` gave me this...

```
from django.conf import settings

from blog.util import whatever

from __future__ import unicode_literals

import datetime

from w0rplib.templatetags.markdown import markdown

from .managers import ArticleManager
```

Even if I set the default ordering as specified in the documentation, the modules appear in some different order.

**Comment by pahko** [comment]
I have exactly the same issue, :+1:

**Comment by ychab** [comment]
:+1:  thanks for a fix

**Comment by dirkhusemann** [comment]
Yep, not working at all:

```
[settings]
multi_line_output=3
known_susi=SuSi
known_anchor=anchor
known_behave=behave
forced_separate=SuSi
sections=FUTURE,ANCHOR,BEHAVE,STDLIB,THIRDPARTY,FIRSTPARTY,SUSI,LOCALFOLDER
```

turns the **expected order**
```python
from __future__ import absolute_import, print_function

import anchor

from behave import given, then, when

from SuSi.cherrypyext import StackedDocRoot
from SuSi.device.service.datarecord.config import DataRecordServiceConfig
from SuSi.utils.SuSiConfigParser import SuSiConfigParser as ConfigParser
```

into

```python
from behave import given, then, when

from __future__ import absolute_import, print_function

import anchor

from SuSi.cherrypyext import StackedDocRoot
from SuSi.device.service.datarecord.config import DataRecordServiceConfig
from SuSi.utils.SuSiConfigParser import SuSiConfigParser as ConfigParser
```

**Comment by danpalmer** [comment]
:+1: for this. How on earth did this pass the test suite?!

**Comment by w0rp** [comment]
These things happen some times, I suppose.

**Comment by danpalmer** [comment]
Any movement on this? I'd quite like to adopt isort on our project, but given that it's essentially randomly sorting sections, there's no advantage to it at the moment.

**Comment by mjtamlyn** [comment]
There is a fixing PR at #293

**Comment by ychab** [comment]
PR merged, issue fixed !

**Comment by timothycrosley** [comment]
Fixed in version 4.1.0

Thanks everyone!

~Timothy
